src/site/content/en/blog/model-viewer/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: 'The <model-viewer> web component'
+subhead: Adding 3D models to a website can be tricky; &lt;model-viewer&gt; is as easy as writing HTML.
+authors:
+  - joemedley
+date: 2019-02-06
+updated: 2020-08-21
+hero: hero.jpg
+description: |
+  Adding 3D models to a website is tricky for a variety of reasons including
+  the hosting issues and the high bar of 3D programming. That's why we're
+  introducing the &lt;model-viewer&gt; web component to let you use 3D models
+  declaratively.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - 3d
+  - model-viewer
+---
+
+Adding 3D models to a website is tricky. 3D models ideally will be shown in
+a viewer that can work responsively on all browsers including smartphones,
+desktop, or even the new head-mounted displays. The viewer should support
```

Comment:
```suggestion
desktop, or even new head-mounted displays. The viewer should support
```
Seems to read better if you omit the definite article (because the other items in the list don't use a definite article)